.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This looks like an unintentional addition.

Reply:
It's not the first time this has happened (`git log --stat -S ".idea"`). Maybe we should add `.idea` to the .gitignore.